api/v1alpha1/applicationvolumereplication_types.go

```diff
@@ -17,35 +17,43 @@ limitations under the License.
 package v1alpha1
 
 import (
+	plrv1 "github.com/open-cluster-management/multicloud-operators-placementrule/pkg/apis/apps/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Action that will be taken by Ramen when a subscription is paused
+// DRAction that will be taken by Ramen when a subscription is paused
 // +kubebuilder:validation:Enum=Failover;Failback
-type Action string
+type DRAction string
 
-// These are the valid values for Action
+// These are the valid values for DRAction
 const (
 	// Failover, restore PVs to the new home cluster, and unpause subscription
-	ActionFailover Action = "Failover"
+	ActionFailover DRAction = "Failover"
 
 	// Failback, restore PVs to the original home cluster, and unpause subscription
-	ActionFailback Action = "Failback"
+	ActionFailback DRAction = "Failback"
 )
 
-// DREnabledSubscriptionsMap defines the action used for failover per subscription.
-// Key is subscription name. Value is either empty, 'failover', or 'failback'
-type DREnabledSubscriptionsMap map[string]Action
+// ClusterPeersReference holds a reference to ClusterPeers
+type ClusterPeersReference struct {
+	// `namespace` is the namespace of the cluster peers.
+	Namespace string `json:"namespace"`
```

Comment:
Good question. I am tempting to say namespaced so that they disappear when the app disappears.  On the other hand, if it is cluster scope, then when one is created for an app, it would also be for 100s.  I'll note it and discuss it later.